### PR TITLE
New Formula: Libggz

### PR DESCRIPTION
**Pull request by s172262**
This is retry of the #532 and #534 .

### CODE REVIEW

File: libggz.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
+class Libggz < Formula
+  desc "The libggz library wraps many common low-level functions."
+  homepage "http://dev.ggzgamingzone.org/libraries/libggz/"
+  url "http://mirrors.ibiblio.org/ggzgamingzone/ggz/0.0.14.1/libggz-0.0.14.1.tar.gz"
+  sha256 "54301052a327f2ff3f2d684c5b1d7920e8601e13f4f8d5f1d170e5a7c9585e85"
+  # Target: libggz.2.3.0.dylib (version 6.0.0)
```
Comment (dunn) [inline libggz.rb]: You can remove this comment.

File: libggz.rb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  # Libggz of this version is unable to build with gnutls-30 and later.
+  depends_on "Libgcrypt" # Libgcrypt is indispensable to make net-clients.
```
Comment (dunn) [inline libggz.rb]: Should be lowercase; `depends_on "libgcrypt"`.

File: libggz.rb [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}",
+                          "--with-gcrypt"
+    system "make", "check"
+    system "make", "install"
+  end
+
+  test do
+    system "file", "#{HOMEBREW_PREFIX}/lib/libggz.2.3.0.dylib"
```
Comment (dunn) [inline libggz.rb]: We really need a test here that uses the library; take a look at [`libtiff.rb`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/libtiff.rb) and [`tinyxml.rb`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/tinyxml.rb) for examples.

**Comment by dunn** [comment]
Could you squash this into one commit?  Thanks!

**Comment by apjanke** [comment]
@BrewTestBot test this please

**Comment by apjanke** [comment]
Merged. Thank you for your contribution to Homebrew!

**Pull request closed without merging**